recipe/set_bld_opts.bat

```diff
@@ -45,6 +45,7 @@ set BLD_OPTS=%WIN64% ^
     HDF5_DIR=%LIBRARY_PREFIX% ^
     HDF5_LIB=%LIBRARY_LIB%\hdf5.lib ^
     HDF5_H5_IS_DLL=YES ^
+    ILI_ENABLED=YES ^
```

Comment:
I don't know the first thing about this driver but we should probably add a test for it.

Reply:
Colombia is using it as a "standard"  https://github.com/SwissTierrasColombia/LADM_COL

Reply:
If you can come up with some data to test it or check the current tests we have: https://github.com/conda-forge/gdal-feedstock/blob/master/recipe/run_test.py

Reply:
looks like there are some public examples here https://www.interlis.ch/en/downloads/testdatensaetze

Reply:
Cool! And the files are small enough to just use them here. Can you add that test?